guides/oidc/oidc.md

```diff
@@ -0,0 +1,116 @@
+# OpenID Connect
+
+The OpenID Connect (OIDC) feature allows third-party applications to verify the identity of the end-user and to obtain basic user profile information. Instead of managing usernames and passwords directly **trento-web console will rely on an identity provider (IDP) to authenticate users**.
+
+## Enabling OIDC
+
+The OIDC feature is **disabled by default**.
```

Comment:
Again, not OIDC feature. OIDC integration/authentication/usage, or Authentication with OIDC